gradle/fips.gradle

```diff
@@ -5,9 +5,11 @@ import org.elasticsearch.gradle.testclusters.TestDistribution
 // Common config when running with a FIPS-140 runtime JVM
 if (BuildParams.inFipsJvm) {
 
+  String javaSecurityFilename = BuildParams.runtimeJavaDetails.startsWith('Oracle') ? 'fips_oracle_java.security' : 'fips_java.security'
```

Comment:
I took a simpler approach that duplicates the `.security` file. It should also be possible to add a line of ` # security.provider.5=SunRsaSign` into the existing file and uncomment it if the runtime java is oracle. But my gradlefu is not great enough to tell exactly where this should fit. Suggestions are welcome.

Reply:
We already do something similar in 7.x for Zulu 8, I think it s  fine